sdk/search/Azure.Search.Documents/src/AzureSearchDocumentsEventSource.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Tracing;
+using Azure.Core.Diagnostics;
+
+namespace Azure.Search.Documents
+{
+    [EventSource(Name = EventSourceName)]
+    internal sealed class AzureSearchDocumentsEventSource : EventSource
+    {
+        private const string EventSourceName = "Azure-Search-Documents";
```

Comment:
@pakrym - is there a guideline around how we name these?

Reply:
From https://azure.github.io/azure-sdk/dotnet_implementation.html#logging
 

- DO set EventSource name to package name replacing . with - (i.e. . Azure-Core for Azure.Core package)